src/components/Payroll/PayrollOverview/PayrollOverviewPresentation.tsx

```diff
@@ -620,37 +633,137 @@ export const PayrollOverviewPresentation = ({
 
                 return <GenericBlocker key={blockerType} blocker={blocker} />
               })}
-            <Heading as="h3">{t('payrollSummaryTitle')}</Heading>
-            <DataView
-              label={t('payrollSummaryLabel')}
-              columns={[
-                {
-                  title: t('tableHeaders.totalPayroll'),
-                  render: () => <Text>{formatCurrency(totalPayroll)}</Text>,
-                },
-                {
-                  title: t('tableHeaders.debitAmount'),
-                  render: () => (
-                    <Text>{formatCurrency(Number(payrollData.totals?.companyDebit ?? 0))}</Text>
-                  ),
-                },
-                {
-                  title: t('tableHeaders.debitAccount'),
-                  render: () => <Text>{bankAccount?.hiddenAccountNumber ?? ''}</Text>,
-                },
-                {
-                  title: t('tableHeaders.debitDate'),
-                  render: () => <Text>{dateFormatter.formatShortWithYear(expectedDebitDate)}</Text>,
-                },
-                {
-                  title: t('tableHeaders.employeesPayDate'),
-                  render: () => (
-                    <Text>{dateFormatter.formatShortWithYear(payrollData.checkDate)}</Text>
-                  ),
-                },
-              ]}
-              data={[{}]}
-            />
+            <Heading as="h3">
+              {isWireFunds
+                ? t('payrollSummaryTitleWire')
+                : isFourDayDirectDeposit
+                  ? t('payrollSummaryTitleFourDay')
+                  : t('payrollSummaryTitle')}
+            </Heading>
+
+            {isWireFunds ? (
+              <DataView
+                label={t('payrollSummaryLabel')}
+                columns={[
+                  {
+                    title: t('tableHeaders.totalPayroll'),
+                    render: () => <Text>{formatCurrency(totalPayroll)}</Text>,
+                  },
+                  {
+                    title: t('tableHeaders.wireAmount'),
+                    render: () => {
+                      const metadata = selectedUnblockOption?.metadata as
+                        | { companyDebit?: string }
+                        | undefined
+                      const wireAmount = metadata?.companyDebit ?? payrollData.totals?.companyDebit
+                      return <Text>{formatCurrency(Number(wireAmount ?? 0))}</Text>
+                    },
+                  },
+                  {
+                    title: t('tableHeaders.wireTransferDeadline'),
+                    render: () => {
+                      const metadata = selectedUnblockOption?.metadata as
+                        | { wireTransferDeadline?: string }
+                        | undefined
+                      const wireDeadline =
+                        metadata?.wireTransferDeadline ?? payrollData.payrollDeadline
```

Comment:
I think this one should be `metadata?.wireInDeadline`

<img width="433" height="289" alt="Image" src="https://github.com/user-attachments/assets/5b1879c9-74a4-4b0c-ad87-93bd7b72c78d" />

Same feedback here, we should likely not default this to the payrollData.payrollDeadline since that would be misleading since this represents the wire in deadline